dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBEncoderDecoder.java

```diff
@@ -101,6 +102,10 @@
  * Utility functions for JAXB.
  */
 public final class JAXBEncoderDecoder {
+
+    private static final Logger LOG = LogUtils.getLogger(JAXBEncoderDecoder.class);
+    private static boolean isLoggableFinest = LOG.isLoggable(Level.FINEST);  // Liberty change
```

Comment:
Should this be `final`?

Reply:
On further thought, I don't think we should cache this check at the class level. If we do, we can't dynamically update the trace without restarting the server/application.

You could cache it at the start of a trace heavy method if performance is concerned.